src/main/resources/application.conf

```diff
@@ -24,6 +24,11 @@ akka {
   }
 }
 
+api {
+    dev {
+        enabled = true
```

Comment:
Can we move it to hardcode it in the codebase? `application.conf` is quick easy to exchange so one will be able to quickly enable dev endpoints. Ideally we'll use macros to remove it from the codebase, for now I'd set some flag in the constellation node or sth